admin/tests/integration/components/certifications/certification/details-v3_test.js

```diff
@@ -158,23 +158,49 @@ module('Integration | Component | Certifications | certification > details v3',
       assert.deepEqual(expected, result);
     });
 
-    test('displays the modal with the candidate answer on icon click', async function (assert) {
-      // given
-      const store = this.owner.lookup('service:store');
-      this.model = store.createRecord('v3-certification-course-details-for-administration', {
-        certificationChallengesForAdministration: createChallengesForAdministration(['ok', 'ko'], store),
+    module('when the candidate answers several questions', function () {
+      module('when the candidate gives an answer to the question', function () {
+        test('displays the modal with the candidate answer on icon click', async function (assert) {
+          // given
+          const store = this.owner.lookup('service:store');
+          this.model = store.createRecord('v3-certification-course-details-for-administration', {
+            certificationChallengesForAdministration: createChallengesForAdministration(['ok', 'ko'], store),
+          });
+
+          // when
+          const screen = await render(hbs`<Certifications::Certification::DetailsV3 @details={{this.model}} />`);
+
+          const modalButtons = screen.getAllByRole('button', { name: 'Afficher la réponse du candidat' });
+          await click(modalButtons[0]);
+
+          // then
+          const modal = within(await screen.findByRole('dialog'));
+          assert.dom(modal.getByRole('heading', { name: 'Réponse question 1' })).exists();
+          assert.dom(screen.getByText('Réponse 1')).exists();
+        });
       });
 
-      // when
-      const screen = await render(hbs`<Certifications::Certification::DetailsV3 @details={{this.model}} />`);
-
-      const modalButtons = screen.getAllByRole('button', { name: 'Afficher la réponse du candidat' });
-      await click(modalButtons[0]);
-
-      // then
-      const modal = within(await screen.findByRole('dialog'));
-      assert.dom(modal.getByRole('heading', { name: 'Réponse question 1' })).exists();
-      assert.dom(screen.getByText('Réponse 1')).exists();
+      module('when the candidate passes the question', function () {
```

Comment:
```suggestion
      module('when the candidate skips the question', function () {
```
to pass = réussir  
un faux ami sympa